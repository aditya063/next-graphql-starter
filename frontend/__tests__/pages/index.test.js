import { shallow } from 'enzyme';
import IndexPage from '../../pages/index';

describe('IndexPage', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    mockProps = {};
    wrapper = shallow(<IndexPage {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  afterEach(() => jest.resetAllMocks());

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
