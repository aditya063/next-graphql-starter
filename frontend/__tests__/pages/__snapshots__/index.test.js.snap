// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<Fragment>
  <Memo()
    title="Home"
  />
  <Memo() />
</Fragment>
`;
